src/main/resources/bidder-config/adtelligent.yaml

```diff
@@ -18,8 +18,6 @@ adapters:
       supported-vendors:
       vendor-id: 410
     usersync:
-      url: https://sync.adtelligent.com/csync?t=p&ep=0&gdpr={{gdpr}&gdpr_consent={{gdpr_consent}}&us_privacy={{us_privacy}}&redir=
-      redirect-url: /setuid?bidder=adtelligent&gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&us_privacy={{us_privacy}}&uid={uid}
```

Comment:
We prefer to have this properties with empty value, examples can be found in `adhese.yaml`

Reply:
Done